pandas/tests/extension/test_categorical.py

```diff
@@ -195,6 +195,14 @@ def test_searchsorted(self, data_for_sorting):
         if not data_for_sorting.ordered:
             raise pytest.skip(reason="searchsorted requires ordered data.")
 
+    def test_max(self):
+        # GH 24382
+        pass
+
+    def test_min(self):
+        # GH 24382
+        pass
```

Comment:
How come these are empty?

Reply:
The methods added in this PR ignore nan ,i.e., `skipna=True`. The existing categorical.min return nan if categorical contain any nan. This behavior is expected in test_min_max (tests/arrays/categorical/test_analytics.py).

If `min` and `max` of the generic EA ignoring `nan` is what we want, future PR is required to add `skipna` parameter to `categorical.min` and `categorical.max`.

Reply:
We don't want a release where these become out of sync, so perhaps a PR implementing `skipna=True/False` for Categorical first makes sense.